tests/test_inclined_exponential.py

```diff
@@ -30,173 +35,340 @@
     sys.path.append(os.path.abspath(os.path.join(path, "..")))
     import galsim
 
+save_profiles = True
+
 # set up any necessary info for tests
 # Note that changes here should match changes to test image files
 image_dir = './inclined_exponential_images'
 
 # Values here are strings, so the filenames will be sure to work (without truncating zeros)
-fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
-image_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
-image_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
-image_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
-image_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# InclinedExponential test cases, which we'll also test InclinedSersic with
+both_fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
+both_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
+both_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
+both_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
+both_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# InclinedSersic-only test cases
+sersic_fluxes = ("1.0", "2.0", "1.5", "1.1", "10.0", "1.0e6", "1.0e-6", "2.3e4",)
+sersic_ns = ("1.0", "1.5", "2.0", "2.5", "1.0", "1.5", "2.0", "2.5",)
+sersic_inc_angles = ("0.0", "0.0", "0.0", "0.0", "0.1", "0.2", "0.3", "0.5",)
```

Comment:
I can add in a test for the edge-on case. The hold-up here is with generating galaxies with Lance's code. It's pretty inefficient, and I have to go to a much larger size to properly simulating galaxies with n > 1 (since the half-light radius is much higher if the scale radius is kept the same). I think there's also an issue with his code due to the fact that it handles pixel integration through subsampling and summing up subsampled pixels, which means it's likely a bit off near the centre.

All this is to say, it might take a fair bit more computer time to get comparison models which we can trust to percent-level accuracy. I can run some over the holidays, but it would be good to make sure the set of parameters is good enough before doing so, so I don't have to repeat it. How about I keep what's here, but triple all the inclinations to cover a better range (up to 1.5), and add in one more with inclination of pi/2? Should that cover it?